src/features/Rockets/Rockets.css

```diff
@@ -0,0 +1,76 @@
+.rockets {
+  display: flex;
+  flex-direction: column;
+}
+
+.rocketimg {
+  height: 150px;
+  aspect-ratio: 4 / 3;
+}
+
+.rocketTile {
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  margin-bottom: 8px;
+}
+
+.reserved {
+  margin-right: 10px;
+  font-size: medium;
+  background-color: rgb(51, 87, 219);
+  color: white;
+  width: fit-content;
+  width: -moz-fit-content;
+  padding: 6px;
+  border-radius: 8px;
+  font-weight: bolder;
+}
```

Comment:
- [ ] The `Reserved` and `Reserve Rocket` buttons don’t follow the design, kindly make sure your color match to the original design 🙃 

Your design

<img width="1036" alt="Screenshot 2023-05-18 at 16 31 29" src="https://github.com/Amazinggracee/rocket-mission/assets/74501523/c20baf14-dcff-4e4d-835d-75b575d409e1">

Original design

<img width="698" alt="Screenshot 2023-05-18 at 16 41 44" src="https://github.com/Amazinggracee/rocket-mission/assets/74501523/7163c4f6-0139-4c74-92d9-c8ce29eb0cf1">